streams/src/main/java/org/apache/kafka/streams/processor/To.java

```diff
@@ -65,4 +67,25 @@ public To withTimestamp(final long timestamp) {
         this.timestamp = timestamp;
         return this;
     }
+
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        final To to = (To) o;
+        return timestamp == to.timestamp &&
+            Objects.equals(childName, to.childName);
+    }
+
+    /**
+     * Equality is implemented in support of tests, *not* for use in Hash collections, since this class is mutable.
+     */
+    @Override
+    public int hashCode() {
+        throw new UnsupportedOperationException("To is unsafe for use in Hash collections");
```

Comment:
Ok. If a test does not pass, this method is actually called. Hence., if we don't implement it, we don't get a "yellow" test failure with proper error message, but the test crashed "red". \cc @vvcephei